snappy/click.go

```diff
@@ -481,6 +483,129 @@ func removePackageBinaries(m *snapYaml, baseDir string) error {
 	return nil
 }
 
+// this is a bit confusing, a localestring in xdg is:
```

Comment:
Should this logic be in something like a desktop.go file?  click.go seems to be used as a catch-all at the moment.